File: JavaScript/fibonacci.js
function fibonacci() {
    
    n = prompt();

    array = [0, 1];

    console.log("Hello HacktoberFest!");

    for (let i = 0; i < n; i++) {
        if (i > 1) {
            number = array[i-2] + array[i-1];
            array.push(number);
        } else {
            number = array[i];
        }
        console.log(number + " ");
    }
    
}

File: Kotlin/Fibonacci.kt
fun fibo(n: Int): Int {
    return if (n < 2) n else fibo(n - 1) + fibo(n - 2)
}

fun main(args: Array<String>) {
    println("Hello Hacktoberfest")
    print("Digite o número de n: ")
    val n = readLine()!!.toInt()
    for (i in 0..n)
        print(" ${fibo(i)}")
}
